steps/browserstack-espresso-app-automate/0.2.0/step.yml

```diff
@@ -0,0 +1,190 @@
+title: browserstack-espresso-app-automate
```

Comment:
Title is an user facing string. Should be in a format similar to `Send Telegram Message`. The displayed strings can be tested by running `bitrise :workflow-editor` in a directory that has a bitrise.yml referencing the step; which is repository root in this case.

Reply:
Please do a new share/new PR if changing the step.yml, to make sure the commit hash is updated and it references the correct version tag.

Reply:
Got it.  I'll clean that up and open a new PR.